packages/server/src/modules/messages/infra/http/routes/template.ts

```diff
@@ -15,6 +15,11 @@ const templateRouter = express.Router();
 
 templateRouter.use(ensureAuthenticated);
 
+const schema = Yup.object().shape({
+  title: Yup.string().required(),
+  content: Yup.string().required(),
```

Comment:
Maybe we could check for the content having `{{ message_content }}` and remove it from this line: https://github.com/Rocketseat/umbriel/blob/master/packages/server/src/modules/messages/services/CreateTemplateService.ts#L14